Plots/Contours/NCL_h_lat_7.py

```diff
@@ -0,0 +1,231 @@
+"""
+NCL_h_lat_7.py
+==============
+This script illustrates the following concepts:
+   - Drawing vectors over filled contours
+   - Drawing pressure and height scales
+   - Interpolate to user specified pressure levels
+   - Using the geocat-comp method `interp_hybrid_to_pressure <https://geocat-comp.readthedocs.io/en/latest/user_api/generated/geocat.comp.interp_hybrid_to_pressure.html#geocat.comp.interp_hybrid_to_pressure>`_
+   - Using a different color scheme to follow `best practices <https://geocat-examples.readthedocs.io/en/latest/gallery/Colors/CB_Temperature.html#sphx-glr-gallery-colors-cb-temperature-py` for visualizations
```

Comment:
```suggestion
   - Using a different color scheme to follow `best practices <https://geocat-examples.readthedocs.io/en/latest/gallery/Colors/CB_Temperature.html#sphx-glr-gallery-colors-cb-temperature-py>`_ for visualizations
```
This makes the Sphinx format the hyperlink correctly. You can generate the html page to check if things are formatted the way you want going to the `GeoCAT-examples/docs` directory and running `make html`. This will build all of the html pages, which takes a while the first time. After that, it only remakes the files that have been changed since the last run.

Reply:
Thank you!